charts/README.md

```diff
@@ -0,0 +1,63 @@
+# Deploy the Alert Logic Agent Container for Kubernetes using Helm 2
+
+This directory contains the al-agent Chart directory, which has the kubernetes manifests to deploy.
+
+## Before You Begin
+- You must have the kubectl and helm command line interfaces installed and get authentication credentials to interact with the cluster where you want to install the Agent Container.
+- Download the al-agent chart directory.
+- To deploy the Alert Logic Agent Container for Kubernetes, you need the unique registration key for your Alert Logic account.
+
+**To find your unique registration key:**
+1. In the Alert Logic console, click the Support Information icon.
+2. Click "Details."
+3. Copy your unique registration key.
+
+## Deploy the Agent Container
+**To deploy the Agent Chart to your cluster:**
+1. In the command line, type ```kubectl get pods``` to ensure kubectl communicates with the proper Kubernetes cluster.
+2. In the command line, type ```helm upgrade --install al-agent al-agent --set registration_key="${your_registration_key}"```.
```

Comment:
Remove the `--set registration_key` flag or make it optional under a separate step.

Reply:
done